src/features/threadPruner/getActiveThreads.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  APIThreadChannel,
+  Client,
+  RESTGetAPIGuildThreadsResult,
+  Routes,
+} from 'discord.js'
+
+export async function getActiveThreads(
+  client: Pick<Client, 'rest'>,
+  guild: { id: string },
+) {
+  const result = (await client.rest.get(
+    Routes.guildActiveThreads(guild.id),
+  )) as RESTGetAPIGuildThreadsResult
+  return result.threads as APIThreadChannel[]
```

Comment:
Is this one type safe, as we already case the type of `result`?

Reply:
comment added krub